django/website/main/templates/base.html

```diff
@@ -24,9 +24,11 @@ <h1 class="logo"><a href="/">{% settings_value "SITE_NAME" %}</a></h1>
         {% block logframe_switching_form %}
         {% if request.user and request.user.is_authenticated %}
         <form id="logframes-switch" action="{% url "switch-logframes" %}" method="post">{% csrf_token %}
-            <select id="logframes-switch-select" name="logframe" onchange="this.form.submit();">
+            <select id="logframes-switch-select" name="logframe" onchange="this.form.submit();"{% if logframe_list|length == 1 and not perms.logframe.edit_logframe %} disabled="disabled"{% endif %}>
```

Comment:
If there's one Logframe and they don't have permission to create new ones, you end up with an empty drop-down, unless you disable the select box.